Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -323,6 +339,7 @@ public string GetSessionTicket()
                 return sessionTicketString;
             }
 
+#if GET_STEAM_SESSION_TICKET_LEGACY
```

Comment:
We should add or include documentation making clear what this scripting define does (ie why do we have this conditional here?)